src/utils/helper/index.js

```diff
@@ -0,0 +1,19 @@
+import dayjs from "dayjs";
+import { v4 as uuidv4 } from "uuid";
+import { TASK_DATE_FORMAT } from "utils/const";
+
+export const generateTaskObject = (title) => {
+  return {
+    id: uuidv4(),
+    title: title,
+    createdAt: dayjs().format(TASK_DATE_FORMAT),
```

Comment:
avoid using a third party library to format date, use Date API instead.

// see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString